src/Symfony/Bridge/PhpUnit/bin/simple-phpunit

```diff
@@ -15,14 +15,17 @@
 
 error_reporting(-1);
 
-if (PHP_VERSION_ID >= 70200) {
-    // PHPUnit 6 is required for PHP 7.2+
+if (PHP_VERSION_ID >= 70100) {
+    // PHPUnit 7 is required for PHP 7.1+
+    $PHPUNIT_VERSION = getenv('SYMFONY_PHPUNIT_VERSION') ?: '7.4';
+} elseif (PHP_VERSION_ID >= 70000) {
+    // PHPUnit 6 is required for PHP 7.0+
```

Comment:
That comment is not accurate, PHPUnit 5 officially also supports php 7.0 and 7.1. The old comment was correct.

Again, I agree that we should default to PHPUnit 6 on PHP 7.0.

Reply:
Changed